### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by frankgh**
Add debugging information regarding JProfiler

### CODE REVIEW

File: dev/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Debugging
```
Comment (denalex) [inline dev/README.md]: technically this is not debugging, so maybe call these "Profiling"

Comment (frankgh) [reply dev/README.md]: done

File: dev/README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Profiling
+
+### Visual VM Debugging
```
Comment (denalex) [inline dev/README.md]: here too :)

File: dev/README.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+To perform memory profiling add the following line to PXF's environment settings (`pxf/conf/pxf-env.sh`) on the machine where we want to debug:
+
+```
+export CATALINA_OPTS="-Dcom.sun.management.jmxremote=true -Dcom.sun.management.jmxremote.rmi.port=9090 -Dcom.sun.management.jmxremote.port=9090 -Dcom.sun.management.jmxremote.ssl=false -Dcom.sun.management.jmxremote.authenticate=false -Dcom.sun.management.jmxremote.local.only=false -Djava.rmi.server.hostname=127.0.0.1"
+```
+
+### JProfiler Debugging
```
Comment (denalex) [inline dev/README.md]: here too :)

File: dev/README.md (hunk 2 above)
Comment (frankgh) [reply dev/README.md]: done

File: dev/README.md (hunk 3 above)
Comment (frankgh) [reply dev/README.md]: done

**Pull request merged**